packages/ag-charts-community/src/chart/series/cartesian/areaSeries.ts

```diff
@@ -707,6 +701,7 @@ export class AreaSeries extends CartesianSeries<
 
         const fns = prepareAreaPathAnimation(newData, oldData, this.processedData?.reduced?.diff);
         if (fns === undefined) {
+            animationManager.skipCurrentBatch();
```

Comment:
Would this present issues when using on combined series types charts?

Reply:
So it was always the intent that if one series didn't animate, that the others shouldn't either - so I believe this is desirable and consistent with that idea?